build/repo.targets

```diff
@@ -40,18 +40,18 @@
     <Exec Command="dotnet run $(VerifierArgs)" WorkingDirectory="$(VerifierProjectDirectory)" />
   </Target>
 
-  <Target Name="RunSplitPackages" DependsOnTargets="RunResolvePackageDependencies">
+  <Target Name="RunSplitPackages" DependsOnTargets="RunResolvePackageDependencies;_FindDotNetPath">
     <ItemGroup>
       <SplitPackagePath Include="%(PackageDefinitions.ResolvedPath)" Condition="'%(PackageDefinitions.Name)'=='SplitPackages'" />
     </ItemGroup>
     <PropertyGroup>
       <ProductPackagesSource>$(ArtifactsDir)product-packages</ProductPackagesSource>
       <SplitPackagesCsv>$(RepositoryRoot)packages\packages.csv</SplitPackagesCsv>
-      <SplitPackagesBinary>@(SplitPackagePath)\SplitPackages.exe</SplitPackagesBinary>
+      <SplitPackagesBinary>@(SplitPackagePath)\SplitPackages.dll</SplitPackagesBinary>
```

Comment:
Can be simplified using [Item functions](https://msdn.microsoft.com/en-us/library/ee886422.aspx)
```xml
<SplitPackagesBinary>@(PackageDefinitions->WithMetadataValue('Name','SplitPackages')->Metadata('ResolvedPath'))\SplitPackages.dll</SplitPackagesBinary>
```